docs/2.9. Usage - Fully Automated Testing of NMOS Controllers.md

```diff
@@ -0,0 +1,174 @@
+# Testing NMOS Controllers
+
+## Fully Automated NMOS Controller Tests
+
+The testing tool supports fully automated testing of NMOS Controllers by replacing the Testing Façade with your own application to receive and process the requests from the NMOS Testing Tool. A full example utilising Flask and Selenium can be found on the nmos-js repository under AutomatedTesting. 
+
+## Building your fully automated test interface
+
+The Testing Façade IP/Hostname and Port on the test suite should be set to the location of your application.
+POST requests are sent from the Test Suite to the API endpoint `{IP/hostname}:{Port}/x-nmos/testing-facade` 
```

Comment:
Ha, ha! I mentioned the one-word _testingfacade_ source directory... and now I'll ask for the opposite... NMOS URL path segments are typically _not_ hyphenated, like _channelmapping_ and _transportfile_. Yes, this is a different "rule" than that for JSON attributes, e.g. `transport_file`!

Reply:
OK, noted.